src/ssl.c

```diff
@@ -239,6 +239,45 @@ byte tsip_rootCAverified( );
 #endif
 
 #ifdef WOLFSSL_SESSION_EXPORT
+/**
```

Comment:
The doxy style comments for API can be removed since they are in `doc/dox_comments/header_files/ssl.h`. Otherwise we are duplicating language that needs to be updated.